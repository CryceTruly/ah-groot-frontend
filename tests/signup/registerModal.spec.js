import React from 'react';
import { mount, shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import SignupModal from '../../src/components/signup/SignupModal';

const props = {
  open: true,
  close: jest.fn(),
  userSignupRequest: jest.fn(),
  inputHandler: jest.fn(),
  submitHandler: jest.fn(),
  password: 'password',
  email: 'example@gmail.com',
  username: 'username',
};
describe('SignupModal', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<SignupModal {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
