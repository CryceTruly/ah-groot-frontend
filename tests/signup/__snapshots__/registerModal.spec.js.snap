// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupModal should render without crashing 1`] = `ShallowWrapper {}`;
